rust/src/wallet.rs

```diff
@@ -0,0 +1,65 @@
+use crate::seed::Bip39Seed;
+use anyhow::bail;
+use anyhow::Result;
+use bdk::bitcoin::Network;
+use bdk::blockchain::ElectrumBlockchain;
+use bdk::database::MemoryDatabase;
+use bdk::electrum_client::Client;
+use bdk::KeychainKind;
+use bdk::SyncOptions;
+use bdk::Wallet;
+use std::str::FromStr;
+
+pub const MAINNET_ELECTRUM: &str = "ssl://blockstream.info:700";
+pub const TESTNET_ELECTRUM: &str = "ssl://blockstream.info:993";
+
+#[derive(Debug)]
+pub struct WalletInfo {
+    pub phrase: Vec<String>,
+}
+
+pub fn init_wallet(network: &str) -> Result<WalletInfo> {
```

Comment:
For me it is unclear if this API is what we actually want. It feels awkward to instantiate an Electrum client on every call, I think we need different lifecycle management here. 

@klochowicz is working on the lifecycle and state management of the rust components within Dart. I would focus on the UI and not try to achieve the *actual* backend call yet but plug in dummy values for now. 

You could even go as far as implementing a value provider (e.g. factory) that can be switched from dummy value to an actual API call easily.